### DISCUSSION TITLE
[Feature Request]: Switch config enable/disable NVM

### DISCUSSION THREAD

**Discussion by tupywebteam** [beyondcode/herd-community#350] [Ideas]
### Feature Description

After updating herd, I had the inconvenience of having my settings modified in .zshrc

I use asdf and I have no intention of having nvm, so it is interesting that there is the option to enable/disable nvm in the herd

The best case would be to be able to choose between nvm or asdf

### Is this feature valuable for other users as well and why?

Customization setup

**Comment by danieldunderfelt** [comment]
I use fnm, and while Herd is a good way to install PHP, I really don't appreciate that it also forces nvm into my environment. This should be optional, and if node is already installed somehow, Herd shouldn't touch it or replace it.
